torch/csrc/jit/graph_executor.cpp

```diff
@@ -202,6 +208,19 @@ struct GraphExecutorImpl {
     return implementation.run(std::move(inputs));
   }
 
+  std::shared_ptr<Graph> graphFor(const variable_tensor_list& inputs) const {
```

Comment:
Can you use `getOrCreateAutogradFallback()` and `compileSpec` so that this debug function works even if those particular versions have never been run before?